src/backends/kubernetes/kubernetes_backend.py

```diff
@@ -26,7 +26,7 @@
 POD_PREFIX = "tank"
 BITCOIN_CONTAINER_NAME = "bitcoin"
 LN_CONTAINER_NAME = "ln"
-LOGGING_NAMESPACE = "warnet-logging"
+MAIN_NAMESPACE = "warnet"
```

Comment:
in 95610e44f38e2210d93fdffac73cc51a0ae0ed97 Add prometheus to logging stack:

this is added in the previous commit then removed here?